src/_P039_Thermosensors.ino

```diff
@@ -423,20 +423,38 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
           }
           {
             addFormNote(F("Set Thermocouple type for MAX31856"));
-            const String Toptions[10]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T"), F("VM16"), F("VM32") };
+            const String Toptions[10]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T"), F("VM8"), F("VM32") };
+
+            // 2021-05-17: c.k.i.: values are directly written to device register for configuration, therefore no linear values are used here
+            // MAX 31856 datasheet (page 20):
+            //    Thermocouple Type
+            //    0000 = B Type
+            //    0001 = E Type
+            //    0010 = J Type
+            //    0011 = K Type (default)
+            //    0100 = N Type
+            //    0101 = R Type
+            //    0110 = S Type
+            //    0111 = T Type
+            //    10xx = Voltage Mode, Gain = 8. Code = 8 x 1.6 x 217 x VIN
+            //    11xx = Voltage Mode, Gain = 32. Code = 32 x 1.6 x 217 x VIN
+            //    Where Code is 19 bit signed number from TC registers and VIN is thermocouple input voltage
```

Comment:
This should also be in the documentation.

Reply:
Yes, and new configuration parameter averaging as well...already on my personal list - i will add it as feature to the list here.

Reply:
Documentation has been updated with latest features for MAX 31856 within push d908816